pihole-google.txt

```diff
@@ -7753,4 +7753,365 @@ www.tofu.jibemobile.com
 www.tokyo.jibemobile.com
 www.viper.jibemobile.com
 www.watermelon.jibemobile.com
-yellowstone.jibemobile.com
\ No newline at end of file
+yellowstone.jibemobile.com
+# More
+discord-attachments-uploads-prd.storage.googleapis.com
+cs.opensource.google
+gnpfesdk-pa.googleapis.com
+voilatile-pa.googleapis.com
+android-context-data.googleapis.com
+appsgrowthpromo-pa.clients1.google.com
+appsgrowthpromo-pa.clients2.google.com
+appsgrowthpromo-pa.clients3.google.com
+appsgrowthpromo-pa.clients4.google.com
+appsgrowthpromo-pa.clients5.google.com
+appsgrowthpromo-pa.clients6.google.com
+accounts-cctld.l.google.com
```

Comment:
```suggestion
```

Can you remove this domain, see https://github.com/nickspaargaren/no-google#warning-important-regex-filters-not-included-in-blocklist. There are way to much of these to add in the list. We recommend regex for this one.